pkgs/by-name/pa/package-assistant/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "package-assistant";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "patryk-s";
+    repo = pname;
+    rev = version;
+    hash = "sha256-Pd5ONAqqBaOaxF5/SvtE9lNvweCzzbjC7fVj+IKTIoQ=";
+  };
+
+  cargoHash = "sha256-Ct8XvFIj6R8ZMG4s98rO0vfqpKln8Y3Vhhe78a0h9Bo=";
+
+  meta = {
+    description = "Manage your package managers";
+    longDescription = ''
+      Provides a consistent CLI interface for all supported package managers,
+      across multiple OSes, so you don't have to remember the specific syntax on a given system.
+    '';
+    homepage = "https://github.com/patryk-s/package-assistant";
```

Comment:
Its not my decision to make, but as this is quite an early stage project it might well not get picked up for nixpkgs. NUR may be a better option?

Reply:
that's a fair point, thanks.

Does it make sense to leave this PR open *and* go the NUR route at the same time?

Reply:
I don't think there is any harm in doing that, but I don't know the correct course of action for sure.

My guess is that someone with the commit bit will pass by and either merge this, or definitively say they wont, before too long. You could leave it open for a few weeks to see how it turns out rather than duplicating your efforts in NUR immediately